rules/alerting.go

```diff
@@ -413,6 +413,15 @@ func (r *AlertingRule) ActiveAlerts() []*Alert {
 	return res
 }
 
+// IsAlertUnknown returns whether an alert's RuleHealth is still unkown.
+func (r *AlertingRule) IsAlertUnknown() RuleHealth {
```

Comment:
Would `IsHealthUnknown` be better here? Also with such function name, I would be expecting `true` or `false`.

Reply:
@codesome sounds correct